lib/getPaths.js

```diff
@@ -4,21 +4,39 @@
 */
 "use strict";
 
-module.exports = function getPaths(path) {
+const path = require("path");
+
+const isSubDir = function(parent, dir) {
+	const relative = path.relative(parent, dir);
```

Comment:
path.relative is too slow. Can you implement this with `dir.startsWith(parent)` and add a `/` resp `\` to the jail option (in ResolverFactory)?